cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -75,10 +78,11 @@ fi
 #############
 # VARIABLES #
 #############
-YQ_CMD_DELETE=(yq delete -i)
-YQ_CMD_MERGE=(yq merge --inplace --overwrite --append)
-YQ_CMD_WRITE=(yq write --inplace)
-OP_SDK_CMD=(operator-sdk olm-catalog gen-csv --csv-version)
+YQ_CMD_DELETE=($yq delete -i)
+YQ_CMD_MERGE_OVERWRITE=($yq merge --inplace --overwrite --append)
+YQ_CMD_MERGE=($yq merge --inplace --append)
+YQ_CMD_WRITE=($yq write --inplace)
+OP_SDK_CMD=($operator_sdk olm-catalog gen-csv --csv-version)
```

Comment:
should this command be `operator-sdk` (hyphen instead of an underscore)?

Reply:
this looks right. that's a variable pointing to the operator-sdk binary's path.